### PR TITLE
Commit for colab notebooks

### PR DESCRIPTION
**Pull request by PrajwalPrashanth**

### CODE REVIEW

File: jovian/utils/colab.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+
+_colab_file_id = None
+
+
+def in_colab():
+    return 'google.colab' in sys.modules
+
+
+def set_colab_file_id(colab_file_id):
```
Comment (sidujjain) [inline jovian/utils/colab.py:17]: You can use `file_id` as argument here

File: jovian/utils/colab.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import sys
+from jovian.utils.api import _h, get_current_user
```
Comment (sidujjain) [inline jovian/utils/colab.py:2]: imports need to be sorted

File: jovian/utils/colab.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    global _colab_file_id
+    _colab_file_id = colab_file_id
+
+
+def get_colab_file_id():
+    global _colab_file_id
+    return _colab_file_id
+
+
+def perform_colab_commit(project, privacy):
```
Comment (sidujjain) [inline jovian/utils/colab.py:27]: TOL: this won't be capturing any hyperparams/metrics

File: jovian/utils/commit.py [hunk 4]
```diff
@@ -122,6 +123,20 @@ def commit(message=None,
 
     is_cli = kwargs.get('is_cli', False)
 
+    # To commit colab notebooks
+    if in_colab():
```
Comment (sidujjain) [inline jovian/utils/commit.py:127]: can we move this block near kaggle commit?

File: jovian/utils/colab.py (hunk 3 above)
Comment (PrajwalPrashanth) [reply jovian/utils/colab.py:27]: Have created a ticket for follow up task

File: jovian/utils/commit.py (hunk 4 above)
Comment (PrajwalPrashanth) [reply jovian/utils/commit.py:127]: We will need add additional checks to 3 blocks move this near kaggle commit

**Pull request merged**